pages/account/manage/events.js

```diff
@@ -38,9 +38,20 @@ export async function getServerSideProps(context) {
   };
 }
 
+const alerts = {
+  createOrUpdate: {
+    type: "success",
+    message: "Event Created/Updated Successfully",
+  },
+  delete: {
+    type: "success",
+    message: "Event Deleted Successfully",
+  }
+}
+
 export default function ManageEvents({ events }) {
   const router = useRouter();
-  const { success } = router.query;
+  const { success, deleted } = router.query;
```

Comment:
Oh you made the change already, I was still reviewing 😂 

Thank you for the contribution and making the change. I was going to add I think it would be good to also change this line to be `alert` as I think it will simplify lines `64-69`